### PR TITLE
Product listing page

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**
Added product listing page 

Live link  :-    https://suspicious-yonath-3b1e58.netlify.app/productlisting

### CODE REVIEW

File: productListing.html [hunk 1]
```diff
@@ -145,7 +145,104 @@ <h4 class="products_sidebar_title">Rating</h4>
...
+                    </button>
+                  </div>
+                </div>
+        
+                <div class="card-basic card_custom_width">
+                  <div class="badge-container">
+                    <img
+                      class="card-img"
+                      src="https://pyxis.nymag.com/v1/imgs/4a7/29b/4ef2f518834dd81d8ceb374caa54fd5171-22-plant-gifts.rsquare.w700.jpg"
+                    />
```
Comment (VLeads) [inline productListing.html:188]: Add alt tag to img

**Review by VLeads — APPROVED** [review]
Code is clean. And UI is awesome!!

File: styles/productListing.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  padding: 1rem 2rem 1rem 1rem;
+  margin-right: 2rem;
+}
+
+.productlisting_products {
+  width: 75%;
+}
+
+.products_filter_section {
+  display: flex;
```
Comment (vishal804) [inline styles/productListing.css]: Can see that you are having flex Utility Classes Can't we use it here 
Just asking If it is Possible

**Review by vishal804 — APPROVED** [review]

File: styles/productListing.css (hunk 2 above)
Comment (vishal804) [reply styles/productListing.css]: Everything Else Looks Good

File: productListing.html (hunk 1 above)
Comment (abhisheksahani-nova) [reply productListing.html:188]: Did the changes, thanks for the feedback

File: styles/productListing.css (hunk 2 above)
Comment (abhisheksahani-nova) [reply styles/productListing.css]: Did those changes, thanks for the feedback

**Pull request merged**